src/routes/__layout.svelte

```diff
@@ -30,4 +30,10 @@
 	main {
 		padding: var(--s-10) var(--s-5) var(--s-20);
 	}
+
+  @media (min-width: 1280px) {
```

Comment:
lint is failing. I think you need to run `npm run format` to convert these spaces to tabs

Reply:
Oh gotcha let me fix that. I'm on windows, does that affect the tabs/spaces?